src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonEditorView.ts

```diff
@@ -529,11 +526,18 @@ export class ExecutionPlanComparisonEditorView {
 class AddExecutionPlanAction extends Action {
 	public static ID = 'ep.AddExecutionPlan';
 	public static LABEL = localize('addExecutionPlanLabel', "Add execution plan");
-	constructor() {
+
+	constructor(
+		@IAdsTelemetryService private readonly telemetryService: IAdsTelemetryService
+	) {
 		super(AddExecutionPlanAction.ID, AddExecutionPlanAction.LABEL, addIconClassName);
 	}
 
 	public override async run(context: ExecutionPlanComparisonEditorView): Promise<void> {
+		this.telemetryService
+			.createActionEvent(TelemetryKeys.TelemetryView.ExecutionPlan, TelemetryKeys.TelemetryAction.AddExecutionPlan)
```

Comment:
You can just call `sendActionEvent` directly if you aren't adding custom properties